types/vector/fundamental/README.md

```diff
@@ -0,0 +1,13 @@
+# `std::vector<std::int32_t>`
+
+## Fields
+
+ * `[Split]Index{32,64}`
+
+with the corresponding column type for the collection parent field.
```

Comment:
Do you mean payload (child) field? Parent field is the one containing the offset column.

Reply:
No, I mean the offset column - hopefully clarified. The idea was that we don't need the full product of all column encodings for the payload (child) field because that's already tested in `types/fundamental/`.

Reply:
Ah, I understand what you mean now :)